app/logic/Mail/OWA/OWAAccount.ts

```diff
@@ -150,32 +124,36 @@ export class OWAAccount extends MailAccount {
     await ensureLicensed();
     await super.login(interactive);
     await this.loginCommon(interactive);
-    this.authorizationHeader = await appGlobal.remoteApp.OWA.getAnyScrapedAuth(this.partition);
     this.hasLoggedIn = true;
     await this.listFolders();
 
-    // `listFolders()` will subscribe to new user-added calendars
-
-    // Link (until #155) or create the default address book.
-    // TODO: Support user-added address books. Compare addressbook ID.
-    let haveAddressbook = appGlobal.addressbooks.find(addressbook => addressbook.mainAccount == this);
-    if (!haveAddressbook) {
-      let addressbook = newAddressbookForProtocol("addressbook-owa") as OWAAddressbook;
-      addressbook.initFromMainAccount(this);
-      await addressbook.save();
-      appGlobal.addressbooks.add(addressbook);
-    }
-
-    for (let addressbook of appGlobal.addressbooks) {
-      if (addressbook.mainAccount == this) {
-        addressbook.listContacts()
-          .catch(this.errorCallback);
+    let response = await this.callOWA(new OWAGetPeopleFiltersRequest());
+    for (let filter of response) {
```

Comment:
How about renaming `filter` to, like, `ab`?

What is a "filter" (as in `GetPeopleFiltersRequest`)? Aren't we looking for address books? Or does OWA somehow call that "filter?